### PR TITLE
Add task to sync welcome audio files over sftp

### PR DESCRIPTION
**Pull request by erikh360**

### CODE REVIEW

File: vas2nets/views.py [hunk 1]
```diff
@@ -47,7 +47,6 @@ class SyncWelcomeAudioView(APIView):
         vas2nets sftp folder
     """
     def post(self, request, *args, **kwargs):
-        print 'post'
```
Comment (OMosimege) [inline vas2nets/views.py]: This looks great :) looks like the print line was meant to be deleted

File: vas2nets/views.py [hunk 2]
```diff
@@ -37,3 +38,23 @@ def post(self, request, *args, **kwargs):
...
+
+
+class SyncWelcomeAudioView(APIView):
+
+    """ SyncWelcomeAudio Interaction
+        POST - starts up the task that sync the welcome audio files with the
+        vas2nets sftp folder
+    """
+    def post(self, request, *args, **kwargs):
+        print 'post'
```
Comment (OMosimege) [inline vas2nets/views.py]: This looks good cake. 
But like the previous review, there will be a few comments, and questions from my side just for learning experience

Comment (OMosimege) [inline vas2nets/views.py]: Do you want this print to be here? or for the method to just return the response

**Review by OMosimege — COMMENTED** [review]
On a side note, is there a reason travis is failing?

File: vas2nets/views.py (hunk 2 above)
Comment (erikh360) [reply vas2nets/views.py]: I had it there while testing, but I removed it.

**Comment by erikh360** [comment]
@OMosimege you can click on the little red cross and it will show you the travis log, this one probably failed because flake8 picked up the `print` which I later removed. As long as the last commit passes.

Thanks for looking, please give both a thumbs up if you don't see anything wrong. 

Remember that it still need to go through QA so if you miss something it's not a issue.

File: vas2nets/views.py (hunk 2 above)
Comment (OMosimege) [reply vas2nets/views.py]: Awesome, this looks good then

**Review by OMosimege — APPROVED** [review]

**Pull request merged**